Frameworks/UIKit.Xaml/ActivityIndicatorView.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ActivityIndicatorView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root">
+
+    <!-- TODO: Update property names in paths for binding once they have been updated in the header. See #2607 -->
+    <Canvas x:Name="indicatorCanvas"
+            Height="{Binding ElementName=Root, Path=ActivityIndicatorView_Height, Mode=OneWay}"
+            Width="{Binding ElementName=Root, Path=ActivityIndicatorView_Width, Mode=OneWay}"
+            HorizontalAlignment="Center"
+            VerticalAlignment="Center">
+        <ProgressRing x:Name="progressRing"
```

Comment:
Probably worth a large comment as to why we have an extra Canvas (so we can opt out of Grid's clipping).  Too bad Grid doesn't have an option to explicitly opt out of its clipping behavior. :'(